[DOCFIX] Update new contributor doc (#16820)
Alluxio/alluxio: docs/en/contributor/Contributor-Getting-Started.md

larger tasks. This is a good way to familiarize yourself with the entire process of contributing to
the Alluxio project.

> Browse any of the open [New Contributor Alluxio Tasks](https://github.com/Alluxio/alluxio/labels/task-good-first-issue)

Reviewer: give them info about different categories as well?
the different labels (task-good-first-issue, task-easy, task-medium, task-hard)
otherwise they may be scared away by the thousands of issues lol

Author: this would point then to only issues with task-good-first-issue

issue itself. In order to do that you have to add certain text in the pull request description.

> For example, if your issue number is `#123`, you should include one of the following in your
> pull request description.

Reviewer 2: ```suggestion
For example, if your issue number is `#123`, the PR description should include `Fix new contributor task Alluxio/alluxio#123`.
```

also facepalm to this now deleted list of options

Author: List of options makes people confused(at least me facepalm)
